cmd/bb_runner/main.go

```diff
@@ -18,6 +18,7 @@ func main() {
 	var tempDirectoriesList util.StringList
 	var (
 		buildDirectoryPath = flag.String("build-directory", "/worker/build", "Directory where builds take place")
+		listenNetwork      = flag.String("listen-network", "unix", "Network type to listen on: tcp, tcp4, tcp6, unix or unixpacket.")
```

Comment:
The examples given here are not exhaustive, right? Even if the current version of Go only supports these, the actual list is not something we can influence. Maybe better to put 'e.g.' somewhere in the description to reflect that.

Reply:
The latest version of Buildbarn would require no more (or at least less) patching on this end. It is possible to now use UNIX sockets or TCP sockets for any purpose. As in, bb-worker -> bb-runnner can now use TCP as well.